draft-ietf-core-oscore-groupcomm.md

```diff
@@ -842,11 +855,15 @@ The value of the 'kid' parameter in the 'unprotected' field of response messages
 
 The value of the 'kid context' parameter in the 'unprotected' field of request messages MUST be set to the ID Context, i.e., the Group Identifier value (Gid) of the group. That is, unlike in {{RFC8613}}, the 'kid context' parameter is always present in requests.
 
-## AEAD Nonce # {#sec-cose-object-aead-nonce}
+## Nonce Computation # {#sec-cose-object-aead-nonce}
+
+The nonce is constructed like in OSCORE, with the difference that step 4 in {{Section 5.2 of RFC8613}} is replaced with:
 
-The AEAD nonce is constructed like in OSCORE, with the difference that step 4 in {{Section 5.2 of RFC8613}} is replaced with:
+A. and then XOR with the X bytes from the Common IV's start, where X is the length in bytes of the nonce.
```

Comment:
s/with the X bytes/with X bytes

Reply:
Fixed in https://github.com/core-wg/oscore-groupcomm/commit/486ca64666db862fd1e9bc0dc367818a6d8bcde7